process/transaction/interceptedTransaction.go

```diff
@@ -100,6 +105,7 @@ func (inTx *InterceptedTransaction) CheckValidity() error {
 func (inTx *InterceptedTransaction) processFields(txBuff []byte) error {
 	inTx.hash = inTx.hasher.Compute(string(txBuff))
 
+	var err error
```

Comment:
var not needed. initialize with :=

Reply:
Golang is not happy with 
```GO
inTx.sndAddr, err := inTx.addrConv.CreateAddressFromPublicKeyBytes(inTx.tx.SndAddr)
```
(assigning to an existing struct member and creating a new var instance on the same line)